src/PopperContent.js

```diff
@@ -162,7 +165,7 @@ class PopperContent extends React.Component {
     return (
       <ReactPopper modifiers={extendedModifiers} {...attrs} component={tag} className={popperClassName}>
         {children}
-        <Arrow className={arrowClassName} />
+        {hideArrow ? null : <Arrow className={arrowClassName} />}
```

Comment:
This syntax is a little nicer `{hideArrow && <Arrow className={arrowClassName} />}`. Looks good otherwise!

Reply:
Okay,  that was a dilemma for me too. I will fix it.